### DISCUSSION TITLE
fvm v3 vscode settings.json sdk path slashes

### DISCUSSION THREAD

**Discussion by jibbers42** [leoafarias/fvm#634] [General]
After updating to v3 on Windows, my vscode settings.json config changed from
`"dart.flutterSdkPath": ".fvm/flutter_sdk",`
to 
`"dart.flutterSdkPath": ".fvm\\versions\\3.16.9",`

Will the change in slashes cause issues for teams that commit settings.json?

I don't have quick access to a *nix machine at the moment so I don't know if this an issue. There's a good chance it will just work, but I suppose it could break. A more annoying issue is that it may cause .git churn on this line.

I'm curious if it should always use forward slashes?

**Comment by leoafarias** [comment]
@jibbers42 I believe this is correct. However, do you want to test to see if using the POSIX style separator? That would have to be done through Dart Code extension, most likely, but if it does work, I think we can normalize under POSIX

**Reply by jibbers42** [reply]
If I understand what you want me to test - I just verified using `/` works using the following steps:

- `fvm use 3.16.8` (my global install is 3.16.9, so I wanted to be sure vscode wasn't using that)
- in settings.json, changed `"dart.flutterSdkPath": ".fvm\\versions\\3.16.8"` to `"dart.flutterSdkPath": ".fvm/versions/3.16.8"`
- closed and opened vscode
- verified vscode was using 3.16.8
![image](https://github.com/leoafarias/fvm/assets/797824/79e8b337-55a0-4857-9b3c-c00ff36909ae)

Using POSIX path separators seems to work fine.

**Reply by jibbers42** [reply]
Is the plan to change this to write POSIX style paths, or did I misunderstand? Should I create an issue?

**Reply by leoafarias** [reply]
Yes that is plan. If you can create an issue it would be great.
